infor-design/enterprise-wc
# 941 - Fix angular property binding

**Explain the details for making this change. What existing problem does the pull request solve?**
Fixed to angular where attributes property binding was not work properly for:
- ids-toggle-button
- ids-app-menu
- ids-menu-button
- ids-pager
- ids-text
- ids-toolbar
- Added new `ids-text` example in angular examples repo

**Related github/jira issue (required)**:
Closes #941
Related to #931

**Steps necessary to review your pull request (required)**:
- Pull WC branch ([941-angular-property-binding](https://github.com/infor-design/enterprise-wc/tree/941-angular-property-binding)) and build/run
- Use `npm run publish:link` to push WC branch to local NPM repo for linking
- Pull the latest copy of main from the WC Examples repo
- Pull WC Examples branch ([941-angular-property-binding](https://github.com/infor-design/enterprise-wc-examples/tree/941-angular-property-binding)), or use `main` WC Examples branch after PR ([NG#28](https://github.com/infor-design/enterprise-wc-examples/pull/28)) merged
- Switch to `angular-ids-wc` examples directory, and run `npm i && npm link ids-enterprise-wc`
- Use `npm run start` to run the angular examples demo app
- Navigate to below examples and see all attributes should working properly
- http://localhost:4200/ids-toggle-button/example
- http://localhost:4200/ids-toggle-button/icon-align-right
- http://localhost:4200/ids-app-menu/example
- http://localhost:4200/ids-app-menu/example-footer
- http://localhost:4200/ids-menu-button/example
- http://localhost:4200/ids-menu-button/disabled
- http://localhost:4200/ids-menu-button/display-selected
- http://localhost:4200/ids-pager/example
- http://localhost:4200/ids-pager/list
- http://localhost:4200/ids-pager/sandbox
- http://localhost:4200/ids-text/example

Also smoke test WC demo app
- http://localhost:4300/ids-toggle-button
- http://localhost:4300/ids-app-menu
- http://localhost:4300/ids-menu-button
- http://localhost:4300/ids-pager
- http://localhost:4300/ids-data-grid/pagination-client-side.html
- http://localhost:4300/ids-data-grid/pagination-server-side.html
- http://localhost:4300/ids-data-grid/pagination-standalone.html
- http://localhost:4300/ids-text/typography.html

**Included in this Pull Request**:
